google/cloud/spanner/database_admin_connection_test.cc

```diff
@@ -436,55 +417,167 @@ TEST(DatabaseAdminConnectionTest, UpdateDatabaseGetOperationError) {
 
 /// @test Verify that we can list databases in multiple pages.
 TEST(DatabaseAdminConnectionTest, ListDatabases) {
-  auto mock = std::make_shared<MockDatabaseAdminStub>();
-  Instance in("test-project", "test-instance");
-  std::string const expected_parent = in.FullName();
+  gcsa::Database expected_databases[5];
+  ASSERT_TRUE(TextFormat::ParseFromString(
+      R"pb(
```

Comment:
FYI, some old compilers get confused when using raw strings in the `ASSERT_TRUE` macros, that is why we use a separate `auto constexpr kText = R"pb( ... )pb"` variable.